com.github.vikdevelop.googlemaps.yml

```diff
@@ -0,0 +1,58 @@
+app-id: com.github.vikdevelop.googlemaps
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions: 
+  - org.freedesktop.Sdk.Extension.node16
+command: run-a.sh
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+
+build-options:
+  append-path: /usr/lib/sdk/node14/bin
+  env:
+    NPM_CONFIG_LOGLEVEL: info
+modules:
+  - name: googlemaps
+    buildsystem: simple
+    subdir: main
+    build-commands:
+    - install -D -t /app/main main.js
+    - install -D -t /app/main preload.js
+    - install -D -t /app/main renderer.js
+    - install -D -t /app/main package.json
+    - install -D -t /app/main package-lock.json
+    - install -D -t /app/bin/ ../run-a.sh
+    - install -D -t /app/share/applications data/com.github.vikdevelop.googlemaps.desktop
+    - install -D -t /app/share/icons/hicolor/128x128/apps data/com.github.vikdevelop.googlemaps.png
+    - install -D -t /app/share/metainfo data/com.github.vikdevelop.googlemaps.metainfo.xml
+    sources:
+    - type: archive
+      url: https://github.com/vikdevelop/googlemaps/archive/refs/tags/1.0.1.tar.gz
+      sha256: 1ca5b22ebf7279254883c523c005212e2359d6072f3d2ab72776bbcbb729a666
+      dest: main
+         
+    - type: script
+      dest-filename: run-a.sh
+      commands:
+        - zypak-wrapper.sh /app/main/googlemaps "$@"
+
+  - name: node_modules_google_maps
+    buildsystem: simple
+    build-options:
+       env:
+          XDG_CACHE_HOME: /run/build/googlemaps/flatpak-node/cache
+          npm_config_cache: /run/build/googlemaps/flatpak-node/npm-cache
+          npm_config_offline: 'true'
+    build-commands:
+    - npm install --offline
+    - cp -R node_modules /app/main/node_modules
+    sources:
+      - type: archive
+        url: https://github.com/vikdevelop/googlemaps/releases/download/1.0.1/node-modules_googlemaps.zip
```

Comment:
instead you should use the flatpak-builder-tools to generate a manifest for the dependencies.